src/CMakeLists.txt

```diff
@@ -7,4 +7,7 @@ endif()
 
 target_link_libraries(${PROJECT_NAME} ${CCD_LIBRARIES} ${OCTOMAP_LIBRARIES} ${Boost_LIBRARIES})
 
-install(TARGETS ${PROJECT_NAME} ARCHIVE DESTINATION lib/ LIBRARY DESTINATION lib/)
+install(TARGETS ${PROJECT_NAME}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
It's not necessary on linux systems, but on windows systems the runtime destination is used to determine where to install the .dll.  I usually add all three out of habit, if you're not worried about supporting windows systems I can remove the runtime component.